### PR TITLE
Add separator to mark last turn transition

### PR DESCRIPTION
**Pull request by Neicigam**

### CODE REVIEW

File: src/components/DryRunnerCardLog/index.js [hunk 1]
```diff
@@ -23,27 +25,30 @@ const DryRunnerCardLog = React.memo(props => {
...
+                    src={card.image}
+                    alt={card.name}
+                    onClick={() => setZoomedCard(card)}
+                    data-testid='card-log-image'
+                  />
+                )}
+              </Column>
+              {props.cardsThisTurn === index + 1 && index !== 5 && (
+                <CardLogSeparator />
               )}
```
Comment (KittySparkles) [inline src/components/DryRunnerCardLog/index.js:47]: I don’t think inserting an extra element alongside the columns is necessary. I think we should be able to apply an extra class to all cards that are the first in a turn. Something like:

```js
<Image
  wrapperClassName={[
    'DryRunnerCardLog__image-wrapper',
    props.cardsThisTurn === index + 1 &&
      index !== 5 &&
      'DryRunnerCardLog__image-wrapper--turn',
  ].filter(Boolean)}
  className='DryRunnerCardLog__image'
  src={card.image}
  alt={card.name}
  onClick={() => setZoomedCard(card)}
  data-testid='card-log-image'
/>
```

Then on the CSS:

```css
.DryRunnerCardLog__image-wrapper--turn::after {
  content: '';
  position: absolute;
  right: -0.5em;
  height: 80%;
  top: 50%;
  width: 1px;
  transform: translateY(-50%);
  background: linear-gradient(
    to bottom,
    transparent,
    var(--dark-beige),
    transparent
  );
```

Comment (KittySparkles) [reply src/components/DryRunnerCardLog/index.js:47]: Oh, and for the pseudo element to be able to be absolutely positioned, `.DryRunnerCardLog__image-wrapper` needs to be `position: relative`.

Comment (Neicigam) [reply src/components/DryRunnerCardLog/index.js:47]: Great it looks amazing!!

File: src/components/DryRunnerCardLog/index.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ import Row from '../Row'
 import arrayPad from '../../helpers/arrayPad'
 import './index.css'
 
-const DryRunnerCardLog = React.memo(props => {
+export default React.memo(function DryRunnerCardLog(props) {
```
Comment (KittySparkles) [inline src/components/DryRunnerCardLog/index.js:9]: Thank you! 👍

**Pull request merged**